HerPublicWebsite.UnitTests/BusinessLogic/Services/PendingReferralFilterServiceTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using FluentAssertions.Extensions;
+using HerPublicWebsite.BusinessLogic.Models;
+using HerPublicWebsite.BusinessLogic.Services.RegularJobs;
+using Moq;
+using NUnit.Framework;
+using Tests.Builders;
+using Tests.Helpers;
+using static HerPublicWebsite.BusinessLogic.Models.LocalAuthorityData;
+
+namespace Tests.BusinessLogic.Services;
+
+public class PendingReferralFilterServiceTests
+{
+    private Mock<IDateHelper> mockDateHelper;
+    private PendingReferralFilterService pendingReferralFilterService;
+    private DateTime startOfPreviousMonth;
+
+    [SetUp]
+    public void Setup()
+    {
+        startOfPreviousMonth = 1.February(2024);
+        mockDateHelper = new Mock<IDateHelper>();
+        mockDateHelper.Setup(mdh => mdh.GetStartOfPreviousMonth()).Returns(startOfPreviousMonth);
+        
+        pendingReferralFilterService = new PendingReferralFilterService(mockDateHelper.Object);
+    }
+
+    // if LA is now pending, include
+    [TestCase(true, false, false)]
+    [TestCase(true, false, true)]
+    [TestCase(true, true, false)]
+    [TestCase(true, true, true)]
+    // if LA is not now pending but referral was submitted in the last month to a then pending LA, include
```

Comment:
Comments should start with an upper case letter and end in a full stop.

See comment style here: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#comment-style

The example comments that I added in my mark ups followed that convention. Make sure to match small details like that on styling.